display/ssd1306.h

```diff
@@ -947,6 +947,12 @@ class StandardDisplayController : public DisplayControllerBase<Width, col_t>, pu
   volatile Effect* current_effect_;
 };
 
+// The content of multipropdisplaycontroller.h should go in the respective prop files
+// but it is not yet possible at the moment!
+#ifdef PROPS_MULTI_PROP_H
+#include "multipropdisplaycontroller.h"
```

Comment:
I think what we should do is to put the display controller file in the prop directory.
Then we'll add a define in the prop file, something like:

```cpp
#define EXTRA_DISPLAY_CONTROLLER_INCLUDE "props/multiprop_display_controller.h"
```

Then in ProffieOS.ino, after including display/ssd1306.h, we'll put:

```cpp
#ifdef EXTRA_DISPLAY_CONTROLLER_INCLUDE
#include EXTRA_DISPLAY_CONTROLLER_INCLUDE
#endif
```